### DISCUSSION TITLE
How to Train Your Custom Models with Ultralytics HUB!

### DISCUSSION THREAD

**Discussion by NLadi** [ultralytics/ultralytics#13272] [Show and tell]
Our new blogpost by [Nicolai Nielsen](https://github.com/niconielsen32) is a step-by-step guide on Ultralytics HUB. From training to monitoring your projects, this blog will take you through all you need to know about how to use this user-friendly platform to your advantage.

🔎 Key Highlights: 
✅ Datasets and Models
✅ Training a model 
✅ Advanced features and integrations 

Learn more ➡️ https://www.ultralytics.com/blog/how-to-train-your-custom-models-with-ultralytics-hub

Join our GitHub Discussions to share your thoughts and learn more from the community!

[![Episode 10 Get started with Ultralytics HUB (1) 1](https://github.com/ultralytics/ultralytics/assets/155453586/cfec5759-f28e-423c-bc8b-ecdf1575662c)](https://www.ultralytics.com/blog/how-to-train-your-custom-models-with-ultralytics-hub)

**Comment by pderrenger** [comment]
Thanks for sharing this insightful blog post! It's a great resource for anyone looking to get started with Ultralytics HUB and train their custom models efficiently. The step-by-step guide by Nicolai Nielsen really simplifies the process from setting up datasets to training and monitoring projects.

For those interested in diving deeper, don't forget to check out the advanced features and integrations that can enhance your projects. Join our GitHub Discussions to share your experiences or ask questions. Your contributions help make our community a rich resource for all users! 🚀

[Read the full blog post here.](https://www.ultralytics.com/blog/how-to-train-your-custom-models-with-ultralytics-hub)
